nixos/modules/cctl.nix

```diff
@@ -0,0 +1,69 @@
+{ lib, pkgs, config, ... }:
+let
+  inherit (lib)
+    types
+    mkOption
+    mkIf
+    mkMerge
+    mkEnableOption
+    mdDoc
+    ;
+  cfg = config.services.cctl;
+in
+{
+  options.services.cctl = {
+
+    enable = mkEnableOption (mdDoc "cctl");
+
+    package = mkOption {
+      type = types.package;
+    };
+
+    port = mkOption {
+      type = types.port;
+      default = 11101;
+      example = 60000;
+      description = mdDoc ''
+        Port to listen on.
+        TODO make port configurable in cctl
```

Comment:
Why not just do it now?

Reply:
Because CCTL does not offer a way to do that yet. Mark is aware of it and working on it.